app/Makefile

```diff
@@ -48,7 +48,7 @@ endif
 
 APPVERSION_M=0
 APPVERSION_N=10
-APPVERSION_P=0
+APPVERSION_P=4
```

Comment:
Are we jumping from 0.10.0 to 0.10.4?

Reply:
I am leaving space. We plan to insert a small release containing staking transactions. This will be 0.10.1. I can see 0.10.2 somehow happen. Maybe we should change it to 0.11.0. What do you think?

Reply:
Because of the breaking change?